itango/itango.py

```diff
@@ -30,9 +30,11 @@
 from IPython.core.interactiveshell import InteractiveShell
 from IPython.terminal.ipapp import launch_new_instance
 
-try:     # IPython 4.x
+ipython_version = IPython.__version__
+limit_version = '4.0.0'
+if ipython_version > limit_version : # IPython > 4.x
```

Comment:
Shouldn't this should be `>=`, since 4.0.0 is using `traitlets`?

There is also a more subtle problem here.  Doing string comparison doesn't always work with semantic versioning.  The `packaging` package can help with this:
```shell
In [1]: limit_version = '4.0.0'

In [2]: example_version = '10.0.0'

In [3]: example_version >= limit_version
Out[3]: False

In [4]: from packaging import version

In [5]: limit_version = version.parse('4.0.0')

In [6]: limit_version
Out[6]: <Version('4.0.0')>

In [7]: example_version = version.parse('10.0.0')

In [8]: example_version
Out[8]: <Version('10.0.0')>

In [9]: example_version >= limit_version
Out[9]: True
```

And remember to add `packaging` to the `install_requires` field in `setup.py`.

Reply:
Look my result:

    In [1]: from packaging import version

    In [2]: limit_version = version.parse('4.0.0')

    In [3]: import IPython

    In [4]: ipython_version = IPython.__version__

    In [5]: limit_version
    Out[5]: <Version('4.0.0')>

    In [6]: ipython_version
    Out[6]: '3.2.1'

    In [7]: ipython_version >= limit_version
    Out[7]: True

I think that we need to change the way to get the ipython version.

Reply:
@aureocarneiro In your last line you are comparing a string (`ipython_version`), with a `packaging.version.Version` object (`limit_version`).  We need them to both be "Version" objects.

Use the `version.parse` method to create a "Version" object from the ipython version string:
(I have a different version of ipython, but I used your version at the end)
```
In [1]: from packaging import version

In [2]: limit_version = version.parse('4.0.0')

In [3]: import IPython

In [4]: ipython_version = version.parse(IPython.__version__)

In [5]: limit_version
Out[5]: <Version('4.0.0')>

In [6]: ipython_version
Out[6]: <Version('5.8.0')>

In [7]: ipython_version > limit_version
Out[7]: True

In [8]: ipython_version = version.parse('3.2.1')

In [9]: ipython_version
Out[9]: <Version('3.2.1')>

In [10]: ipython_version > limit_version
Out[10]: False
```

Reply:
Done ! commits: 028a94c and bff3f80